TaskManager.js

```diff
@@ -119,8 +119,45 @@ class TaskManager {
    findTaskById(taskId) {
       return this.tasks.find((task) => task.id === taskId);
    }
-}
 
+   showExpiredTasks() {
+      
+      const currentDate = new Date();
+      const expiredTasks = this.tasks.filter((task) => !task.completed && task.deadline && new Date(task.deadline) < currentDate);
+      
+      console.log('Expired tasks:');
+      
```

Comment:
Where condition, when empty list for expired tasks?
Think about it. 🤓

Reply:
Okey